### PR TITLE
chore: add ci and docker builds

### PR DESCRIPTION
**Pull request by sidohaakma**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+            when {
+                changeRequest()
+            }
+            environment {
+                TAG = "PR-${CHANGE_ID}"
+                DOCKER_CONFIG="/root/.docker"
+            }
+            steps {
+                container('node') {
+                    sh "npm lerna run build --since origin/master"
```
Comment (fdlk) [inline Jenkinsfile:67]: main

**Comment by sidohaakma** [comment]
[CI](https://trello.com/c/8EVWSaVq/584-ci)

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+RUN mkdir -p /server/src
+WORKDIR /server
+
+COPY packages/server/node_modules /server/node_modules
+RUN rm -rf /server/node_modules/armadillo-auth-ui
+RUN mkdir -p /server/node_modules/armadillo-auth-ui/dist
+COPY packages/client/dist /server/node_modules/armadillo-auth-ui/dist
+COPY packages/server/src/ /server/src
+
+CMD ["node", "src/index.js", "NODE_ENV=production"]
```
Comment (fdlk) [inline Dockerfile:12]: Don't put that in the command but add ENV statement on previous line

File: packages/server/.env.example [hunk 3]
```diff
@@ -1,9 +1,17 @@
...
 
+# FusionAuth URL
 ISSUER_BASE_URL='https://auth.molgenis.org/'
+# ClientID that corresponds with the application registered in the FusionAuth server
 CLIENT_ID = '789b9417-ca2b-4707-ab9c-471373d91ba0'
+# ClientSECRET that corresponds with the application registered in the FusionAuth server
 CLIENT_SECRET = 'xxx'
+# Apollo server base url
 BASE_URL=http://localhost:4000
+# ???
```
Comment (fdlk) [inline packages/server/.env.example:16]: Entropy needed to generate secure session IDs

File: packages/server/package.json [hunk 4]
```diff
@@ -1,20 +1,20 @@
...
+    "dev": "nodemon src/index.js"
   },
   "keywords": [],
   "author": "MOLGENIS",
   "license": "LGPL-3",
   "dependencies": {
     "apollo-datasource-rest": "^0.9.3",
     "apollo-server-express": "^2.17.0",
-    "armadillo-js-permissions": "^0.1.0",
+    "armadillo-auth-ui": "^0.1.0",
```
Comment (fdlk) [inline packages/server/package.json:17]: this still doesn't work so great when you run npm install on a dev machine I think?

Comment (sidohaakma) [reply packages/server/package.json:17]: > this still doesn't work so great when you run npm install on a dev machine I think?

This should work when you use lerna. So 

`npx lerna bootstrap`

and then `npm install` and `npm build` on `armadillo-auth-ui`

`npm install` on `armadillo-auth-server` should work

**Pull request merged**